File: 황창식/C_sharp/Class_practice_20210208/Class_practice_20210208/Car.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Class_practice_20210208
{
    class Car
    {
        // 클래스 상수 변수
        public const string DAERI = "동대구 영업소";

        // 인스턴스 변수
        private string company;
        private string color;
        private string model;
        private string price;

        // 기본 생성자
        public Car()
        {
            /*company = "기아";
            color = "블랙";
            model = "K7";
            price = "4천만원";*/
        }

        // 오버로딩 (중복 정의)
        // 생성자
        public Car(string company, string color, string model, string price)
        {
            this.company = company;
            this.color = color;
            this.model = model;
            this.price = price;
        }

        // 람다식 표현
        public string Company { get => company; set => company = value; }
        public string Color { get => color; set => color = value; }
        public string Model { get => model; set => model = value; }
        public string Price { get => price; set => price = value; }


        /*public string Company
        {
            get { return company; }
            set { company = value; }
        }

        public string Color
        {
            get { return color; }
            set { color = value; }
        }

        public string Model
        {
            get { return model; }
            set { model = value; }
        }

        public string Price
        {
            get { return price; }
            set { price = value; }
        }*/

        public void showCarInfo()
        {
            Console.WriteLine("--------차량 정보--------");
            Console.WriteLine("제조사 : " + company);
            Console.WriteLine("색생 : " + color);
            Console.WriteLine("모델명 : " + model);
            Console.WriteLine("가격 : " + price);
            Console.WriteLine("대리점명 : " + DAERI);
            Console.WriteLine();
        }
    }
}

File: 장승민/Java/210113_classPractice1_review/src/Car.java

public class Car {
	static final String JIJUM="동대구 영업소";
	private String model;
	private String company;
	private String color;
	private String price;
	
	public Car() {
		model = "K7";
		company = "기아";
		color = "블랙";
		price = "4천만원";
	}
	
	public Car(String model, String company, String color, String price) {
		this.model = model;
		this.company = company;
		this.color = color;
		this.price = price;
	}
	
	public void showCarInfo() {
		System.out.println("차량 영업소 :"+JIJUM);
		System.out.println("제조사 : "+ company);
		System.out.println("모델 : "+ model);
		System.out.println("가격 : "+ price);
		System.out.println("색상 : "+ color);
	}
	
}
